src/__init__.py

```diff
@@ -0,0 +1,81 @@
+import os
+from dynaconf import Dynaconf

```

Comment:
@bhavikapanara we don't need all these configuration files for a standalone library.

Reply:
done